integration-testing/examples/jvm-sample/build.gradle.kts

```diff
@@ -0,0 +1,28 @@
+buildscript {
+    val atomicfu_version = rootProject.properties["atomicfu_version"]
```

Comment:
And what is the proper way to get atomicfu_version here?)

Reply:
The error is: `The root project is not yet available for build.`
And I get the same error for version catalogs, so, I only managed to do it like this:

```
dependencyResolutionManagement {
    versionCatalogs {
        create("libs") {
            version("atomicfu", "0.22.0-SNAPSHOT")
            version("kotlin", "1.9.20")
        }
    }
}
```

But it's still versions hardcoding 👀

Reply:
I guess it's fine to leave like it is for now then